docs/project/contributors.md

```diff
@@ -114,163 +80,49 @@ Blocknet is an open-source project with a team consisting of contributors from a
-	<div class="name">Arlyn Culwick&nbsp;&nbsp;
-		<a href="https://www.linkedin.com/in/arlynculwick/" target="_blank"><i class="fa fa-linkedin-square fa-1x" aria-hidden="true"></i></a>
-	</div>
-	<div class="title">Co-Founder</div>
-	<div class="platform">Discord <span class="handle">@synechist#0385</span></div>
-</div>
-
-<div class="contributor">
-	<div class="img-round"><img src="/img/contributors/joe-caterina.jpg"></div>
-	<div class="name">Joe Caterina&nbsp;&nbsp;
-		<a href="https://www.linkedin.com/in/josephcaterina/" target="_blank"><i class="fa fa-linkedin-square fa-1x" aria-hidden="true"></i></a>
-	</div>
-	<div class="title">China Liaison</div>
-	<div class="platform">&nbsp; <span class="handle"></span></div>
-</div>
-
-<div class="contributor">
-	<div class="fa fa-user-circle" aria-hidden="true"></div>
-	<div class="name">Egan Kim&nbsp;&nbsp;
-		<a href="https://www.linkedin.com/in/josephcaterina/" target="_blank"><i class="fa fa-linkedin-square fa-1x" aria-hidden="true"></i></a>
-	</div>
-	<div class="title">Korea Liaison</div>
-	<div class="platform">&nbsp; <span class="handle"></span></div>
-</div>
-
 <div class="contributor">
 	<div class="fa fa-user-circle" aria-hidden="true"></div>
 	<div class="name">@Philip Marshall</div>
-	<div class="title">Social Coordinator</div>
+	<div class="title">Social Media Coordinator</div>
 	<div class="platform">Discord <span class="handle">@Philip Marshall#9143</span></div>
 </div>
 
 <div class="contributor">
 	<div class="fa fa-user-circle" aria-hidden="true"></div>
 	<div class="name">@CRYPTO CORGI</div>
-	<div class="title"><a href="https://twitter.com/BlockDXExchange" target="_blank">Block DX Twitter</a></div>
+	<div class="title"><a href="https://twitter.com/BlockDXExchange" target="_blank">Official Twitter Accounts</a></div>
 	<div class="platform">Discord <span class="handle">@CRYPTO CORGI#6691</span></div>
 </div>
 
-<!-- <div class="contributor">
-	<div class="fa fa-user-circle" aria-hidden="true"></div>
-	<div class="name">@cryptoved</div>
-	<div class="title">Social Media</div>
-	<div class="platform">Discord <span class="handle">@cryptoved#1961</span></div>
-</div> -->
-
-<!-- <div class="contributor">
-	<div class="img-round"><img src="/img/contributors/kerst-bottema.jpg"></div>
-	<div class="name">Kerst Bottema&nbsp;&nbsp;
-		<a href="https://www.linkedin.com/in/kerst-bottema-436b4a26/" target="_blank"><i class="fa fa-linkedin-square fa-1x" aria-hidden="true"></i></a>
-	</div>
-	<div class="title"><a href="https://block-node.info/" target="_blank">block-node.info</a></div>
-	<div class="platform">Discord <span class="handle">@ishkb1#0780</span></div>
-</div> -->
-
-<div class="contributor">
-	<div class="fa fa-user-circle" aria-hidden="true"></div>
-	<div class="name">@littleeagle&nbsp;&nbsp;
-		<a href="https://twitter.com/littleeagle20" target="_blank"><i class="fa fa-twitter-square fa-1x" aria-hidden="true"></i></a>
```

Comment:
Is littleeagle removing himself on account of his moderator role on r/satoshistreetbets?